deployment/ansible/roles/model-my-watershed.spark-jobserver/tasks/main.yml

```diff
@@ -37,7 +37,7 @@
     - Restart Nginx
 
 - name: Pull Spark Job Server container image
-  command: /usr/bin/docker pull quay.io/azavea/spark-jobserver:latest
+  command: /usr/bin/docker pull quay.io/azavea/spark-jobserver:0.6.1
```

Comment:
You're right. Addressed in https://github.com/WikiWatershed/model-my-watershed/commit/2b3c6c0c2294502d28879d160d9a46408c49943c.